src/plone/restapi/profiles/default/rolemap.xml

```diff
@@ -9,6 +9,11 @@
     </permission>
     <permission name="plone.restapi: Access Plone user information" acquire="True">
       <role name="Manager"/>
+      <role name="Site Administrator"/>
```

Comment:
This one is okay, since the permission is defined here in plone.restapi. But we do need an upgrade step. I think it's fine for the upgrade step to also go in plone.restapi.

Reply:
My question is whether this upgrade will not be lost in plone.restapi. Generally when we update Plone, we do not check the upgrade step on a product-by-product. If we run the Plone upgrade, will this upgrade here be run?

Reply:
@wesleybl The perhaps surprising answer is: yes. plone.app.upgrade has a list of specific packages (core addons) whose upgrade steps will also be run when using the @@plone-upgrade view, and it includes plone.restapi: https://github.com/plone/Products.CMFPlone/pull/3457

Reply:
Cool

Reply:
I created the upgrade step.